lib/controllers/frontend/spree/api/v2/storefront/account_controller_decorator.rb

```diff
@@ -0,0 +1,28 @@
+module Spree
+  module Api
+    module V2
+      module Storefront
+        module AccountControllerDecorator
+          def self.prepended(base)
+            base.skip_before_action :require_spree_current_user, only: [:create]
+          end
+
+          def create
+            user = Spree.user_class.new_with_session(spree_user_params, session)
+            if user.save
+              render_serialized_payload { serialize_resource(user) }
+            else
+              render json: { errors: user.errors.messages }
```

Comment:
please use `render_error_payload` from spree - https://github.com/spree/spree/blob/49109826413e48cf84923551da9f6eab9f90a54f/api/app/controllers/spree/api/v2/base_controller.rb#L46

Reply:
@damianlegawiec this method not suitable for requirements. It cannot show multiply errors etc. Do you want me to adjust this method?

Reply:
@chhga it is, please see https://github.com/spark-solutions/spree/blob/223f259b3666ade8fbb06665c970a7134614425b/core/lib/spree/service_module.rb#L26